src/components/Rating.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Container, Header } from 'semantic-ui-react';
```

Comment:
We are not using `Head` which is imported here.

Reply:
This has been removed. Thank you for pointing this out